andela/devrepublic-bn-backend
# #170947570 manager re-confirm request

#### What does this PR do?
add the re-confirm feature where a manager is able to confirm a decision (accepted or rejected) on a request
#### Description of Task to be completed?
- add the reconfirm controller and route
- test the feature
-  add the managerId and confirm attribute to the request table
#### How should this be manually tested?
- clone this repo
- run npm install
-  run the server (npm run dev)
- make a put request to /api/v1/trips/:requestId/confirm
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#170947570
#### Screenshots (if appropriate)
<img width="1049" alt="Screen Shot 2020-02-26 at 16 53 40" src="https://user-images.githubusercontent.com/51261911/75356394-92e61380-58b8-11ea-87ec-dbde8d34072f.png">
<img width="963" alt="Screen Shot 2020-02-26 at 16 53 13" src="https://user-images.githubusercontent.com/51261911/75356416-9da0a880-58b8-11ea-8931-5f5376d0450b.png">
<img width="1028" alt="Screen Shot 2020-02-26 at 16 52 01" src="https://user-images.githubusercontent.com/51261911/75356441-a5f8e380-58b8-11ea-8344-be560e32523a.png">
<img width="1035" alt="Screen Shot 2020-02-26 at 16 51 47" src="https://user-images.githubusercontent.com/51261911/75356498-b315d280-58b8-11ea-9b75-f7cadca9f454.png">

#### Questions:
